pootle/apps/pootle_store/forms.py

```diff
@@ -373,3 +407,129 @@ def save(self, **kwargs):
             super(UnitCommentForm, self).save(**kwargs)
 
     return UnitCommentForm
+
+
+class UnitSearchForm(forms.Form):
+
+    initial = forms.BooleanField(required=False)
+    count = forms.IntegerField(required=False)
+    path = forms.CharField(
+        max_length=2048,
+        required=True)
+    uids = MultipleArgsField(
```

Comment:
To be honest I still don't get why we need to pass `uids`, in plural. But obviously it is not part of this PR to change this.